paseos/actors/spacecraft_actor.py

```diff
@@ -104,6 +107,46 @@ def mass(self) -> float:
         """
         return self._mass
 
+    @property
+    def body_mesh(self) -> np.array:
+        """Gives the mesh of the satellite.
+
+        Returns:
+            np.array: Mesh of the satellite.
+        """
+        return self._spacecraft_body_model._body_mesh
+
+    @property
+    def attitude_disturbances(self) -> list:
+        """Gives attitude disturbances list.
+
+        Returns:
+            list: attitude disturbances list.
+        """
+        return self._attitude_model._disturbances
+
+    @property
+    def body_moment_of_inertia(self) -> np.array:
+        """Gives the moment of inertia of the actor, assuming constant density.
+
+        Returns:
+            np.array: Mass moments of inertia for the actor
+
+        I is the moment of inertia, in the form of [[Ixx Ixy Ixz]
+                                                    [Iyx Iyy Iyx]
+                                                    [Izx Izy Izz]]
+        """
+        return self._spacecraft_body_model._body_moment_of_inertia
+
+    @property
+    def body_center_of_gravity(self) -> np.array:
+        """Gives the volumetric center of mass of the actor.
+
+        Returns:
+            np.array: Coordinates of the center of gravity of the mesh.
+        """
+        return self._spacecraft_body_model._body_center_of_gravity
```

Comment:
need to mention (and I would almost put it in function name) that this is in body frame